ui/css/itcss/tools/utilities.scss

```diff
@@ -1,37 +0,0 @@
-/*
-  Utility Classes
- */
-
-/* lib */
-.flex-column {
-  display: flex;
-  flex-direction: column;
-}
-
-.flex-row {
-  display: flex;
-  flex-direction: row;
-}
-
-.pointer {
-  cursor: pointer;
-}
-
-.cursor-pointer {
-  cursor: pointer;
-  transform-origin: center center;
-  transition: transform 50ms ease-in-out;
-}
-
-.cursor-pointer:hover {
-  transform: scale(1.05);
-}
-
-.cursor-pointer:active {
-  transform: scale(0.97);
-}
-
-.drop-menu-item {
-  display: flex;
-  align-items: center;
-}
```

Comment:
`drop-menu-item` classname is not used

![Screenshot 2025-02-07 at 3 00 07 PM](https://github.com/user-attachments/assets/f1d6ae20-b746-452b-a61f-4c4b933a3081)